po/POTFILES

```diff
@@ -1,6 +1,4 @@
 src/Application.vala
-src/DBus.vala
```

Comment:
This files exists in the repository

Reply:
Yes, but it does not seem to contain any translatable strings

Reply:
Better keep it then so that we get translations if for some reason there is a translatable string here in the future :)